cmd2/cmd2.py

```diff
@@ -175,25 +175,26 @@ def cat_decorator(func):
     return cat_decorator
 
 
-def with_argument_list(func: Callable[[Statement], Optional[bool]],
-                       preserve_quotes: bool = False) -> Callable[[List], Optional[bool]]:
+def with_argument_list(preserve_quotes: bool = False) -> Callable[[List], Optional[bool]]:
```

Comment:
Now that I think about it, it seems like True would be a much better default value for this.  What do you think.  Honestly in my memory I didn’t know this argument existed and I thought we always preserved quotes.